auto_post_classifier/utils.py

```diff
@@ -45,3 +47,36 @@ def check_JSON_format(json_data):
     except jsonschema.exceptions.ValidationError as e:
         logging.error(f"JSON is not valid: {e.message}")
         return False
+
+def generate_score_for_df(post: pd.Series):
```

Comment:
@nilnil47 I'm not sure I get the difference between these two implementations...